toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -326,26 +326,17 @@ chroot_and_install_rpms freetype
 build_rpm_in_chroot_no_install fontconfig
 chroot_and_install_rpms fontconfig
 
-
-# Build OpenJDK and OpenJRE
-echo Java bootstrap version:
+# Download JDK rpms
+echo Download JDK rpms
 case $(uname -m) in
     x86_64)
-        echo $($LFS/usr/lib/jvm/OpenJDK-212-b04-bootstrap/bin/java -version)
-        build_rpm_in_chroot_no_install openjdk8
+        wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/x86_64/msopenjdk-11-11.0.13%2B8-LTS-4.x86_64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
     ;;
     aarch64)
-        echo $($LFS/usr/lib/jvm/OpenJDK-1.8.0.181-bootstrap/bin/java -version)
-        build_rpm_in_chroot_no_install openjdk8_aarch64
+        wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/aarch64/msopenjdk-11-11.0.13%2B8-LTS-4.aarch64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
     ;;
 esac
 
-# Install OpenJDK and OpenJRE
-chroot_and_install_rpms openjdk8
```

Comment:
Out of curiosity, do you know why we installed it before? Just want to make sure nothing was implicitly depending on this being installed.

Reply:
I believe we initially built all of the jdk-dependent packages in the toolchain. I don't believe anything currently in the toolchain has any implicit dependencies though.

Reply:
Confirmed by looking through our old repo. We initially built apache-maven in the toolchain which required the jdk8 install